core/src/com/po/fuck/collections/DrawableCollection.java

```diff
@@ -0,0 +1,38 @@
+package com.po.fuck.collections;
+
+import com.po.fuck.CenterDrawer;
+import com.po.fuck.updates.Drawable;
+
+import java.util.ArrayList;
+
+public class DrawableCollection extends SimpleCollection<Drawable> implements Drawable {
+    public static final int MAX_Z = 20;
```

Comment:
Not related to a PR. This constant could be moved to a constants files, because there are already some logic related constants. Non logic related constants should be moved to the properties.